lib/config/rpcProviderProdConfig.json

```diff
@@ -46,54 +51,61 @@
     "latencyEvaluationSampleProb": 0.25,
     "healthCheckSampleProb": 0.005,
     "providerInitialWeights": [1],
-    "providerUrls": ["QUICKNODE_137"]
+    "providerUrls": ["QUICKNODE_137"],
+    "providerNames": ["QUICKNODE"]
   },
   {
     "chainId": 42161,
     "useMultiProviderProb": 1,
     "latencyEvaluationSampleProb": 0.1,
     "healthCheckSampleProb": 0.002,
     "providerInitialWeights": [1],
-    "providerUrls": ["QUICKNODE_42161"]
+    "providerUrls": ["QUICKNODE_42161"],
+    "providerNames": ["QUICKNODE"]
   },
   {
     "chainId": 8453,
     "useMultiProviderProb": 1,
     "latencyEvaluationSampleProb": 0.05,
     "healthCheckSampleProb": 0.0005,
     "providerInitialWeights": [1],
-    "providerUrls": ["QUICKNODE_8453"]
+    "providerUrls": ["QUICKNODE_8453"],
+    "providerNames": ["QUICKNODE"]
   },
   {
     "chainId": 1,
     "useMultiProviderProb": 1,
     "latencyEvaluationSampleProb": 0.05,
     "healthCheckSampleProb": 0.001,
-    "providerInitialWeights": [1, 0],
-    "providerUrls": ["QUICKNODE_1", "QUICKNODERETH_1"]
+    "providerInitialWeights": [0.01, 0.99],
+    "providerUrls": ["QUICKNODE_1"],
```

Comment:
number of weights have to be equal to the number of provider urls

Reply:
good catch - i even had a test on this that wasn't properly setup